pkgs/applications/editors/lapce/default.nix

```diff
@@ -117,18 +114,9 @@ rustPlatform.buildRustPackage rec {
 
   postInstall = ''
     install -Dm0644 $src/extra/images/logo.svg $out/share/icons/hicolor/scalable/apps/lapce.svg
+    install -Dm0644 $src/extra/linux/dev.lapce.lapce.desktop $out/share/applications/lapce.desktop
```

Comment:
Don't we need to fix this up https://github.com/lapce/lapce/blob/master/extra/linux/dev.lapce.lapce.desktop#L11